### DISCUSSION TITLE
How can i style the dropdown?

### DISCUSSION THREAD

**Discussion by CrispenGari** [azeezat/react-native-select#25] [Q&A] [answered]
How can i style the `dropdown`, I've tried to use the `dropdownStyle` property but it seems like some styles are not being applied to the dropdown, like i want to reduce the `height`, `padding` and `width` of the dropdown but it seems like it's not working. 
```tsx
    <Dropdown
      dropdownStyle={{
        borderWidth: 0, // <--- this works but other properties are not working
      }}
     ....
    />
```

**Comment by azeezat** [comment, marked as answer]
@CrispenGari The `minHeight` set on the component is preventing you from doing this. You can override it as follows. There is an example in the [sandbox](https://azeezat.github.io/react-native-select/)

```js 
    dropdownContainerStyle={{ width: 200  }}
    dropdownStyle={{
        paddingVertical: 5,
        paddingHorizontal: 5,
        minHeight: 40,
      }}
    dropdownIconStyle={{ top: 15, right: 10 }}
```
I hope this helps.

**Reply by CrispenGari** [reply]
Thanks so much, How can i change the `color` of the selected item?

**Reply by azeezat** [reply]
**for single selection**

```js
 selectedItemStyle={{
                  color: 'hotpink',
                  fontWeight: '900',
                }}
```

**For multiple select**

```js
 multipleSelectedItemStyle={{
              borderRadius: 0,
              backgroundColor: 'hotpink',
              color: 'black',
            }}
```

**Reply by CrispenGari** [reply]
You are the best, Is it also possible to change the dropdown color maybe to `red`??

**Reply by azeezat** [reply]
> You are the best, Is it also possible to change the dropdown color maybe to `red`??

Yes, there is are some examples in the [sandbox](https://azeezat.github.io/react-native-select/) https://azeezat.github.io/react-native-select/
